suite/tests/client-interface/cleancall-opt-1.dll.c

```diff
@@ -109,7 +86,18 @@ event_basic_block(void *dc, void *tag, instrlist_t *bb,
      * We use a workaround involving ADR. */
     IF_AARCH64(save_current_pc(dc, bb, entry, &cleancall_start_pc, before_label));
     PRE(bb, entry, before_label);
-    dr_insert_clean_call(dc, bb, entry, func_ptrs[i], false, 0);
+
+    switch (i) {
+    default:
```

Comment:
I copied the switch structure from https://github.com/DynamoRIO/dynamorio/blob/master/suite/tests/client-interface/inline.dll.c#L222